src/core/tests/type_prop/identity.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/Identity.hpp"
+
+#include <gtest/gtest.h>
+
+#include "common_test_utils/test_assertions.hpp"
+#include "common_test_utils/type_prop.hpp"
+#include "openvino/op/constant.hpp"
+
+using namespace testing;
+
+class TypePropIdentityV15Test : public TypePropOpTest<ov::op::v15::Identity> {};
+
+TEST_F(TypePropIdentityV15Test, default_ctor) {
```

Comment:
Add tests with non-default constructor

Reply:
After removing copy param, this is the only constructor available